docs/decisions/0002-footer-links.rst

```diff
@@ -0,0 +1,168 @@
+
+Status
+******
+
+**Draft (=> Accepted)**
+
+.. **Accepted** *2023-05-11*
+
+.. Standard statuses
+    - **Draft** if the decision is newly proposed and in active discussion
+    - **Provisional** if the decision is still preliminary and in experimental phase
+    - **Accepted** *(date)* once it is agreed upon
+    - **Superseded** *(date)* with a reference to its replacement if a later ADR changes or reverses the decision
+
+    If an ADR has Draft status and the PR is under review, you can either use the intended final status (e.g. Provisional, Accepted, etc.), or you can clarify both the current and intended status using something like the following: "Draft (=> Provisional)". Either of these options is especially useful if the merged status is not intended to be Accepted.
+
+Context
+*******
+
+.. This section describes the forces at play, including technological, political, social, and project local. These forces are probably in tension, and should be called out as such. The language in this section is value-neutral. It is simply describing facts.
+
+The footer component is basic - it only has a trademark logo and a language
+selector. It doesn't look similar to the legacy UI footer.
+
+It also doesn't have many customization options. You can customize
+LMS base URL and trade mark logo.
+
+If you want to have more advanced customizations, it's necessary to fork the
+component, modify it, and then install the fork as a package instead of the
+default footer at build time of MFEs.
+
+We want to make the MFE footer to closer resemble the legacy UI footer by
+adding the same footer links as in the legacy UI, and to make it easier to
+style the footer component.
+
+Decision
+********
+
+.. This section describes our response to these forces. It is stated in full sentences, with active voice. "We will …"
+
+Links
+=====
+
+For specifying the footer links, we are going to use `MFE's config mechanism`_.
+That would allow to specify a list of links in various ways:
+
+* At build time:
+  * Via environmental variables
+  * By providing them in ``env.config.js``, like so:
+
+    .. code-block:: javascript
+
+        const config = {
+            FOOTER_LINKS: [
+                {"url": "https://google.com", "text": "Terms of service"},
+                {"url": "https://duckduckgo.com", "text": "Acceptable Use Policy"},
+                {"url": "https://openedx.org", "text": "Privacy Policy"},
+            ],
+        };
+
+        export default config;
+
+* At runtime, via ``MFE_CONFIG`` in LMS's Django settings:
+
+    .. code-block:: python
+
+        MFE_CONFIG = {
+            "FOOTER_LINKS": [
+                {"url": "https://google.com", "text": "Terms of service"},
+                {"url": "https://duckduckgo.com", "text": "Acceptable Use Policy"},
+                {"url": "https://openedx.org", "text": "Privacy Policy"},
+            ],
```

Comment:
Shouldn't the configuration be a bit richer than this? i.e. perhaps support multiple columns, i.e. social links, business links, etc like the current API supports. Each section could potentially have a header. Additionally, an option to show/hide the logo. 

Other than allowing custom links perhaps, this seems like a downgrade from the branding API.

Reply:
@xitij2000 Since I'm rewriting the ADR to use the branding API, I think it will satisfy this.